src/dynamo/s3client.ts

```diff
@@ -2,54 +2,42 @@
 import dotenv from 'dotenv';
 dotenv.config();
 
-import AWS from 'aws-sdk';
+import { S3 } from '@aws-sdk/client-s3';
+import { fromEnv } from '@aws-sdk/credential-providers';
 
-// Load the AWS SDK for Node.js
-
-// Set the region
-AWS.config.update({
-  s3ForcePathStyle: !!process.env.AWS_ENDPOINT,
-  accessKeyId: process.env.AWS_ACCESS_KEY_ID,
-  secretAccessKey: process.env.AWS_SECRET_ACCESS_KEY,
-  region: process.env.AWS_REGION,
-});
-
-const s3 = new AWS.S3({
+export const s3 = new S3({
   endpoint: process.env.AWS_ENDPOINT || undefined,
+  forcePathStyle: !!process.env.AWS_ENDPOINT,
+  credentials: fromEnv(),
+  region: process.env.AWS_REGION,
 });
 
 export const getObject = async (bucket: string, key: string): Promise<any> => {
   try {
-    const res = await s3
-      .getObject({
-        Bucket: bucket,
-        Key: key,
-      })
-      .promise();
+    const res = await s3.getObject({
+      Bucket: bucket,
+      Key: key,
+    });
 
-    return JSON.parse(res!.Body!.toString());
+    return JSON.parse(await res!.Body!.transformToString());
```

Comment:
Functional difference, S3 API now returns Streams instead of Buffers https://docs.aws.amazon.com/sdk-for-javascript/v3/developer-guide/migrate-s3.html#amazon-s3-stream-vs-buffer